packages/gatekeeper/src/components/logout.tsx

```diff
@@ -16,10 +22,27 @@ export const defaultLogoutProps: LogoutProps = {
   redirectPath: LOGIN_URL
 };
 
+/**
+ * Open another window and go to the logout URL.
+ * @param {string} logoutURL URL string representing the auth server logout URL endpoint.
+ */
+function logoutFromAuthServer(logoutURL: string) {
+  const logoutWindow: Window | null = window.open(logoutURL);
```

Comment:
Can this be achieved using a `fetch`?

Reply:
It's a little difficult/challenging because of [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). I'll need some times to say for sure if it can be done or not.

Reply:
Do you think a comment regarding the choice (mainly not using fetch) is something worthwhile?

Reply:
Another thing: we are assuming that logging out will always happen via `GET` requests.  This is not always the case, and in fact many people think that [logging out should be done via POST requests](https://softwareengineering.stackexchange.com/a/196873).  Please add some documentation stating that we only currently support logging out using GET requests.